test/expect/cmd.predefined-macros.stdout.txt.in

```diff
@@ -6,7 +6,7 @@
 #define __castxml_major__ @CastXML_VERSION_MAJOR@
 #define __castxml_minor__ @CastXML_VERSION_MINOR@
 #define __castxml_patch__ @CastXML_VERSION_PATCH@
-#define __clang__ 1(
+#define __clang__ 1\(?
 #define __clang_literal_encoding__ [^
 ]*)?
```

Comment:
Thanks, but this patch makes no sense.  The `(` is not meant to be matched literally.  It corresponds to the `)?` a couple lines below and is meant to make the entire `#define __clang_literal_encoding__` optional.

Also, the test passes for me locally.

Reply:
...without this patch.

Reply:
So, mmm why is not passing tests in the new windows-server 2025?

Reply:
The linked failure shows this extra definition in the actual output:

```
#define __cdecl __attribute__((__cdecl__))
```

I don't see that locally.  Against what version of LLVM/Clang is that job building?

Reply:
In cmakelists is llvm 18

Reply:
#276 adds toleration of that line, but I'd appreciate help tracking down where the `__cdecl` comes from.

Reply:
Thank you! @bradking! Will you release 0.6.11?